src/Oscar.jl

```diff
@@ -198,17 +198,20 @@ function __init__()
         (GAP.Globals.IsSubgroupFpGroup, FPGroup),
     ])
     __GAP_info_messages_off()
+    # make Oscar module accessible from GAP (it may not be available as
+    # `Julia.Oscar` if Oscar is loaded indirectly as a package dependency)
+    GAP.Globals.BindGlobal(GapObj("Oscar"), Oscar)
+    GAP.Globals.SetPackagePath(GAP.Obj("OscarInterface"), GAP.Obj(joinpath(@__DIR__, "..", "gap", "OscarInterface")))
+    GAP.Globals.LoadPackage(GAP.Obj("OscarInterface"))
     withenv("TERMINFO_DIRS" => joinpath(GAP.GAP_jll.Readline_jll.Ncurses_jll.find_artifact_dir(), "share", "terminfo")) do
       GAP.Packages.load("browse"; install=true) # needed for all_character_table_names doctest
     end
     GAP.Packages.load("ctbllib")
     GAP.Packages.load("forms")
     GAP.Packages.load("wedderga") # provides a function to compute Schur indices
     GAP.Packages.load("repsn")
```

Comment:
I guess these could be turned into dependencies for OscarInterface. However, I still want to migrate to `GAP_pkg_repsn` etc. and I am not sure this would be the right thing then... In any case, it could still be done later, if we decide we want to do it after all.

Reply:
We could turn each such call into a suggested package, but I am not sure that this is a good idea.
As far as I see, it would not be possible to load OscarInterface with the OnlyNeeded option, such this change would be safe.
(I think the call `GAP.Packages.load("ferret"; install=true)` in `src/NumberTheory/GaloisGrp/GaloisGrp.jl` cannot be expressed in terms of GAP's dependencies.)